server/eventlog.go

```diff
@@ -135,11 +141,22 @@ func getVerifiedCosState(coscel cel.CEL, pcrs *tpmpb.PCRs) (*pb.AttestedCosState
 			return nil, err
 		}
 
+		// TODO: Add support for post-separator container data
+		if seenSeparator {
+			return nil, fmt.Errorf("found COS Event Type %v after LaunchSeparator event", cosTlv.EventType)
+		}
+
 		switch cosTlv.EventType {
 		case cel.ImageRefType:
+			if cosState.Container.GetImageReference() != "" {
+				return nil, fmt.Errorf("found more than one ImageRef event")
```

Comment:
This can be fmt.Error (no formatting), same w other cases below

Reply:
Unfortunely, [`fmt.Errorf`](https://pkg.go.dev/fmt#Errorf) is the only Error function in the `fmt` package. The alternative would be to use `errors.New`, but staying with `fmt.Errorf` seems fine.

See:

Reply:
I got it confused with t.Error(), looks good then.